jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/ConfigurationPropertyValidator.java

```diff
@@ -66,11 +66,11 @@
     // Warn if a system property is missing
     String missingProperty =
         "%s system property is set, but %s is not; attempting other authentication methods.";
-    if (!commandlinePassword.isEmpty() && commandlineUsername.isEmpty()) {
+    if (!commandlinePassword.isEmpty()) {
       logger.accept(
           LogEvent.warn(String.format(missingProperty, passwordProperty, usernameProperty)));
     }
-    if (!commandlineUsername.isEmpty() && commandlinePassword.isEmpty()) {
+    if (!commandlineUsername.isEmpty()) {
```

Comment:
This makes sense. Then, can we do the same for `auth` checking below? The logic has the same structure.